configure.ac

```diff
@@ -262,6 +262,14 @@ AC_ARG_WITH(system-install-dir,
 SYSTEM_INSTALL_DIR=$with_system_install_dir
 AC_SUBST(SYSTEM_INSTALL_DIR)
 
+AC_ARG_WITH(external-install-dir,
+           [AS_HELP_STRING([--with-external-install-dir=DIR],
+                           [Location of the external installation [LOCALSTATEDIR/endless-extra/flatpak]])],
+           [],
+           [with_external_install_dir='$(localstatedir)/endless-extra/flatpak'])
+EXTERNAL_INSTALL_DIR=$with_external_install_dir
+AC_SUBST(EXTERNAL_INSTALL_DIR)
```

Comment:
It would probably be nicer if this defaulted to not being set (`g_path_is_absolute` would catch the empty string case) rather than encoding the endless default here, but I'll let you wrangle that with upstream. I guess this makes it nicer for Endless developers hacking on flatpak.